docs/reference/buildx_bake.md

```diff
@@ -143,6 +143,11 @@ Use the `-f` / `--file` option to specify the build definition file to use.
 The file can be an HCL, JSON or Compose file. If multiple files are specified,
 all are read and the build configurations are combined.
 
+Alternatively, the environment variable `BUILDX_BAKE_FILE` can be used to specify the build definition to use.
+This is mutually exclusive with `-f` / `--file`; if both are specified, the environment variable is ignored.
```

Comment:
Ok seems it follows compose behavior https://docs.docker.com/reference/cli/docker/compose/#set-up-environment-variables but I would tend to error out in such case. Although with bake we print the loaded definitions during build so at least user is aware of performed definitions.

Reply:
> we print builder used and bake definitions loaded in the logs as "hints"

Incidentally, that is one of the main reasons I deviated from compose's behavior of converting environment-provided files to absolute paths (which I noted in the commit comment).  An absolute path made the output very ugly.  And after seeing the input files were added as trace attributes, I figured not forcing an absolute path would simply avoid the question of whether it would be (or needed to be) scrubbed or not.

Just wanted to call that out in case it triggers any thoughts.  I thought/hoped by putting the processing where I did, the questions/rules about where relative paths are anchored would be no different than for `--file`.

Reply:
Let me know what you think of what I pushed.  Most of the 'good' solutions I could see entailed adding code disproportionate to the feature itself or changing the flow (which you already said you liked).  I left it as a separate commit anticipating you might want (possibly substantial) changes or even opt to drop it.